tests/js/index.js

```diff
@@ -3,34 +3,36 @@ import { expect } from "chai";
 import {
     defaultElementsBondsDataEntry,
     ELEMENT_COLORS,
-    ELEMENT_VDW_RADII,
-    ELEMENTS_BY_SYMBOL,
     filterBondsDataByElementsAndOrder,
     getElectronegativity,
     getElementsBondsData,
     PERIODIC_TABLE,
+    Element,
```

Comment:
We should consider adding tests for the new functionality

Reply:
New tests were added for `getAtomicPropertiesFlat()` and functions of `ChemicalElement`.